### DISCUSSION TITLE
Usage scenario

### DISCUSSION THREAD

**Discussion by kination** [ServerlesslyStack/Serverlessly#119] [General]
Hello~
I could roughly get the concept of project, but still cannot figure out how to use it.

Do you have some plan for usage guide?

**Comment by SachinShekhar** [comment]
Wait. Without at least one Protocol, Serverlessly can't be used.

If you're curious, this is how you'll be able to use it: https://github.com/ServerlesslyStack/Serverlessly/blob/main/%40serverlessly/core/README.md

**Reply by kination** [reply]
@SachinShekhar yes I've check this part.
But I couldn't find a way to make deployment through `awsLambda`, with making sample with
```
import { dbConnect, apiAuth } from './middlewares/essentials';
import { businessLogic1, businessLogic2 } from './middlewares/business';
```

Just let me know if you're working on it
(or I could support making example 👍 )

**Reply by SachinShekhar** [reply]
To create a `Platform Adapter` for AWS Lambda, we first need a `Protocol` like `HTTP` or `WebSocket` (both supported by AWS API Gateway or Load Balancer). I am currently working on `HTTP` which should be finished by the end of this month.
